src/reference/asciidoc/kafka.adoc

```diff
@@ -761,6 +761,28 @@ public KafkaTemplate<String, String> myReplyingTemplate() {
 }
 ----
 
+NOTE: You can annotate a `@KafkaListener` method with `@SendTo` even if no result is returned.
+This is to allow the configuration of an `errorHandler` that can forward information about a failed message delivery to some topic.
+
+[source, java]
+----
+@KafkaListener(id = "voidListenerWithReplyingErrorHandler", topics = "someTopic",
+        errorHandler = "voidSendToErrorHandler")
+@SendTo("failures")
+public void voidListenerWithReplyingErrorHandler(String in) {
```

Comment:
This is a bit unusual behavior and isn't self-explanatory, but having the use-case description here in Docs, I'm fine to merge.

And I will hope no one will follow this pattern. 
That is really much organic to use `KafkaTemplate` in the `ErrorHandler` if we are going to send error result somewhere and our original `@KafkaListener` is just `void`. Just IMO 😄 

Merging...